mkt/site/tasks.py

```diff
@@ -40,15 +42,16 @@ def send_email(recipient, subject, message, real_email, from_email=None,
 
 
 @task
-def set_modified_on_object(obj, **kw):
+def set_modified_on_object(app_label, model_name, pk, **kw):
```

Comment:
pretty sure this doesn't actually need to be a task. Maybe that'd make things simpler if it wasn't?